### ISSUE TITLE
Mozilla accounts work

### ISSUE THREAD

**Issue by Sancus** [thundernest/addons-server#248]
1. Find all references to "Firefox account[s]" and the FxA privacy notice.
2. Change them to "account"
3. Remove any FxA logos (don't think we have any)
4. Update any link to [FxA privacy notice links](https://www.mozilla.org/en-US/privacy/firefox/#firefox-accounts) to the New Link)(forthcoming)

**Comment by MelissaAutumn** [comment]
@Sancus Was point 3 suppose to be "Mozilla accounts"?

**Comment by Sancus** [comment]
> @Sancus Was point 3 suppose to be "Mozilla accounts"?

No, we're in the group that is doing brand-agnostic "rebrand" here like hubs/relay etc, eg:


![image](https://github.com/thundernest/addons-server/assets/479671/4ba00084-2308-45d6-88d7-70ff873209c8)
